ui/apps/platform/src/Containers/VulnMgmt/Reports/Components/ScopeName.tsx

```diff
@@ -1,64 +1,68 @@
 import React, { ReactElement } from 'react';
-import { Button, ButtonVariant, Spinner } from '@patternfly/react-core';
+import { useHistory } from 'react-router-dom';
+import { Alert, Button, ButtonVariant } from '@patternfly/react-core';
 
 import LinkShim from 'Components/PatternFly/LinkShim';
 import { getEntityPath } from 'Containers/AccessControl/accessControlPaths';
 import useFeatureFlags from 'hooks/useFeatureFlags';
-import useFetchScopes from 'hooks/useFetchScopes';
-import useCollection from 'Containers/Collections/hooks/useCollection';
-import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
 import { collectionsBasePath } from 'routePaths';
+import { ReportScope } from 'hooks/useFetchReport';
 
 type ScopeNameProps = {
-    scopeId: string;
+    reportScope: ReportScope | null;
+    // TODO This isn't a prop-type, and it -is- used. Not sure why the lint error needs to be suppressed
```

Comment:
So it looks like this throws a lint error since the prop isn't used in _every_ component that takes this type as a parameter.

Possibly an eslint bug? If not its a questionable rule.

This code will go away next release, so I'm not inclined to investigate to far into it.